config/core/configmaps/gc.yaml

```diff
@@ -60,27 +60,50 @@ data:
     # V2 Garbage Collector Settings
     # ---------------------------------------
     #
-    # These settings are enabled via the 'responsive-revision-gc' feature flag.
+    # These settings are enabled via the "responsive-revision-gc" feature flag.
     # ALPHA NOTE: This feature is still experimental and under active development.
+    #
+    # Stale
+    #   * Revisions are considered stale if they were created after more than
+    #      "retain-since-create-time" or last referenced by a route more than
+    #      "retain-since-last-active-time" ago.
+    #   * Either timeout may be set to "forever" to turn off time-based GC.
+    # Active
+    #   * Revisions which are referenced by a Route are considered active and
+    #     are exempted from GC.
+    #   * Individual revisions may be marked with the annotation
+    #      "knative.dev/no-gc":"true" to be permanently considered active.
+    #
+    # Example config to immediately collect any inactive revision:
+    #    min-stale-revisions: "0"
+    #    retain-since-create-time: "0"
+    #    retain-since-last-active-time: "0"
+    #
+    # Example config to always keep around the last ten revisions:
+    #     retain-since-create-time: "disabled"
+    #     max-non-active-revisions: "10"
+    #
+    # Example disable all GC:
+    #     retain-since-create-time: "disabled"
+    #     max-non-active-revisions: "disabled"
```

Comment:
Reading comments, it seems that just setting first is enough to disable the GC, since a revision would never be considered non-active, hence won't even contest to be in the pool for deletion?

Reply:
I think we want to disable all staleness times. Updated to reflect that.

I did think adding a few examples of some common modes/usages helped make it a little more clear?